src/theme/carousel.ts

```diff
@@ -18,8 +18,8 @@ export default (options: Required<ModuleOptions>) => ({
       vertical: {
         container: 'flex-col -mt-4',
         item: 'pt-4',
-        prev: '-top-12 left-1/2 -translate-x-1/2 rotate-90',
-        next: '-bottom-12 left-1/2 -translate-x-1/2 rotate-90'
+        prev: '-top-12 left-1/2 -translate-x-1/2 rotate-90 rtl:-rotate-90',
```

Comment:
But you only applied rotate for `vertical` mode here 🤔

Reply:
> But you only applied rotate for `vertical` mode here 🤔

@benjamincanac check after update image

![rotate](https://github.com/user-attachments/assets/513dda67-42d0-4d35-95b1-8c04dba137aa)